scripts/e2e/cmd/runner/networking-v1-mfu_one_namespace_one_ingress_test.go

```diff
@@ -94,6 +94,48 @@ var _ = Describe("networking-v1-MFU", func() {
 			Expect(err).To(BeNil())
 		})
 
+		It("[ingress-class] redirect should work after AGIC pod is recreated", func() {
```

Comment:
Why was this test moved?

Reply:
E2E tests are using build flags to execute specific tests that work when AGIC is installed in a specific way.

In the other file, we are testing AGIC in `custom ingress` mode which fails for this test.